Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_InteractTouch.cs

```diff
@@ -299,6 +313,10 @@ private void CreateControllerRigidBody()
 
             var controllerRB = controllerRigidBodyObject.GetComponent<Rigidbody>();
 
+            controllerRigidBodyObject.name = string.Format("[{0}]_RigidBody_Holder", gameObject.name);
```

Comment:
Looks like there's been a bit of a merge conflict here.

The things that have been changed were from a more recent commit, so when the conflict has been resolved, your version has pulled in the older (incorrect version).

Check the method out in the latest master branch and make sure the flow is the same.